open-apparel-registry/open-apparel-registry
# Remove legacy API

## Overview

Remove all references to the legacy, NodeJS API. This includes:

- Docker Compose service definitions
- References in `README.md`
- STRTA

Connects #14 
Resolves #416 

### Checklist


## Testing Instructions

- `./scripts/setup`
- `vagrant ssh -c "./scripts/resetdb"`
- `vagrant ssh -c "./scripts/server"`
- Interact with the application at http://localhost:6543/ and verify things are a-OK.
